pkgs/development/python-modules/bsddb3/default.nix

```
@@ -29,10 +24,18 @@ buildPythonPackage rec {
     export BERKELEYDB_DIR=${pkgs.db.dev};
   '';
 
+  postPatch = ''
+    substituteInPlace test3.py \
+      --replace-warn "from distutils.util import get_platform" "import platform" \
+      --replace-warn "get_platform()" "f'{platform.system().lower()}-{platform.machine()}'" \
+      --replace-warn "_TextTestResult" "TextTestResult"
+  '';
+
   meta = with lib; {
     description = "Python bindings for Oracle Berkeley DB";
     homepage = "https://www.jcea.es/programacion/pybsddb.htm";
-    license = with licenses; [ agpl3Only ]; # License changed from bsd3 to agpl3 since 6.x
+    license = with licenses;
+      [ agpl3Only ]; # License changed from bsd3 to agpl3 since 6.x
```

The comment is not really useful anymore. 

```suggestion
    license = licenses.agpl3Only;
```

---

Done